azerothcore/azerothcore-wotlk
# Fix (DB/gameobject): Band-aid fix for Malygos platform.

Nexus Raid Platform no longer collapses on the first Saronite Bomb use
##### CHANGES PROPOSED:

-  Modify Nexus Raid Platform's health from 100 to 1.500.000, thus making it way harder to break
-  Approach a more elegant solution where the platform doesn't take any damage from siege effects in the first place. // DONE by changing the gameobject faction to 35.


###### ISSUES ADDRESSED:
 

Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1948


##### TESTS PERFORMED:


Tested, the platform now breaks upon taking 1.500.000 damage ( an unreachable ammount by normal standards) rather than on 100 damage.

A saronite bomb does 150 siege damage, 25 players using Saronite Bombs on cooldown and considering 11 saronite bombs can be used before Malygos enters berserk mode (10 min), the maximum damage that will be dealt to the platform will be 41.250, if we add in Global Thermal Sapper Charges -250 siege damage per use, it will still take an unimaginable amount of pulls to actually break the platform if the platform does not recover hp between pulls (not tested yet, i'll keep working on it).
The fix is not perfect and is a band-aid solution , I will probably return with the proper one once I figure a way to make the platform immune to siege damage caused by bombs, Global Thermal Sapper Charge and the likes.

##### HOW TO TEST THE CHANGES:

Enter the Eye of Eternity on 10 or 25 man modes.
.tele nexus or .go creature id 28859 with .gm fly on enabled (you might fall to your death otherwise)
.learn 61766 - Saronite Bomb or .learn 56488  Global Thermal Sapper Charge
Use the learned spell on the platform, notice  that it does take damage but no longer breaks due to the way higher HP threshold. 


##### KNOWN ISSUES AND TODO LIST:


- [x] Make the platform unable to be broken by players accidentally in the first place.
- [x] Changed the platform's faction to 35 from 0.


##### Target branch(es):

Master
